builtin/providers/aws/resource_aws_route53_record.go

```diff
@@ -68,13 +66,10 @@ func resourceAwsRoute53Record() *schema.Resource {
 				ConflictsWith: []string{"alias"},
 			},
 
-			// Weight uses a special sentinel value to indicate it's presense.
-			// Because 0 is a valid value for Weight, we default to -1 so that any
-			// inclusion of a weight (zero or not) will be a usable value
 			"weight": &schema.Schema{
 				Type:     schema.TypeInt,
 				Optional: true,
-				Default:  -1,
```

Comment:
I'm thinking I should leave this line in so that terraform doesn't try to modify every route53 resource already on record since the default value changes from -1 to 0 if this line is removed.